frontend/__tests__/containers/ConfigStep/SourceControl.test.tsx

```diff
@@ -125,6 +125,24 @@ describe('SourceControl', () => {
     expect(screen.getByText(TOKEN_ERROR_MESSAGE[1])).toHaveStyle(ERROR_MESSAGE_COLOR);
   });
 
+  it('should not show error message when field does not trigger any event given an empty value', () => {
+    setup();
+
+    screen.getByTestId('sourceControlTextField').querySelector('input') as HTMLInputElement;
+
+    expect(screen.queryByText(TOKEN_ERROR_MESSAGE[1])).not.toBeInTheDocument();
+  });
+
+  it('should show error message when focus on field given an empty value', () => {
+    setup();
+
+    const tokenInput = screen.getByTestId('sourceControlTextField').querySelector('input') as HTMLInputElement;
+    fireEvent.focus(tokenInput, { target: { value: '' } });
```

Comment:
same issue, see the more issue in the PR, raised by `Codacy Production / Codacy Static Code Analysis`